internal/runners/mcp/rebuildproject/rebuildproject.go

```diff
@@ -0,0 +1,88 @@
+package rebuildproject
+
+import (
+	"encoding/json"
+	"fmt"
+
+	"github.com/ActiveState/cli/internal/captain"
+	"github.com/ActiveState/cli/internal/errs"
+	"github.com/ActiveState/cli/internal/output"
+	"github.com/ActiveState/cli/internal/primer"
+	"github.com/ActiveState/cli/internal/runbits/commits_runbit"
+	"github.com/ActiveState/cli/pkg/platform/authentication"
+	"github.com/ActiveState/cli/pkg/platform/model"
+	bpModel "github.com/ActiveState/cli/pkg/platform/model/buildplanner"
+	"github.com/ActiveState/cli/pkg/project"
+)
+
+type ProjectErrorsRunner struct {
+	auth     *authentication.Auth
+	output   output.Outputer
+	svcModel *model.SvcModel
+}
+
+func New(p *primer.Values) *ProjectErrorsRunner {
+	return &ProjectErrorsRunner{
+		auth:     p.Auth(),
+		output:   p.Output(),
+		svcModel: p.SvcModel(),
+	}
+}
+
+type Params struct {
+	project *project.Namespaced
+}
+
+func NewParams(project *project.Namespaced) *Params {
+	return &Params{
+		project: project,
+	}
+}
```

Comment:
Please make Project public. And nitpick but NewParams() should not take any of the params if we are to stick to the pattern.

When interacting with runners via the CLI (ie. via captain) we use params to pass properties by reference, which we can't do if they're private and only provided as arguments to a constructor function.